From 5cea51f60c9971b55c08605b3813a2bdf03585c8 Mon Sep 17 00:00:00 2001
From: Mathieu Parent <math.parent@gmail.com>
Date: Sat, 28 Sep 2019 11:32:50 +0200
Subject: [PATCH] build: Remove tests for rdchk()

These date back to pre-git e19196110cc9d29c3fb330737507adc364f3bd9a in 1998.

More of these should be removed but the rdchk() test caused a timeout
on an ARM builder in Debian. It might just be a fluke but the tests are
pointless regardless.

See also 16eb80058ca4a34df96388c26736590247b6a3f4 "build: Remove tests for
getdents() and getdirentries()"

Signed-off-by: Mathieu Parent <math.parent@gmail.com>
---
 source3/wscript | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/source3/wscript b/source3/wscript
index 4910ed71d74..e22cb16f3fb 100644
--- a/source3/wscript
+++ b/source3/wscript
@@ -108,7 +108,7 @@ def configure(conf):
     conf.CHECK_FUNCS('getrlimit fsync fdatasync setpgid')
     conf.CHECK_FUNCS('setsid glob strpbrk crypt16 getauthuid')
     conf.CHECK_FUNCS('innetgr')
-    conf.CHECK_FUNCS('initgroups select poll rdchk getgrnam getgrent pathconf')
+    conf.CHECK_FUNCS('initgroups select poll getgrnam getgrent pathconf')
     conf.CHECK_FUNCS('setpriv setgidx setuidx setgroups syscall sysconf')
     conf.CHECK_FUNCS('atexit grantpt posix_openpt fallocate')
     conf.CHECK_FUNCS('fseeko setluid')
@@ -403,7 +403,7 @@ _open __open _opendir __opendir
 pathconf poll
 posix_memalign pread _pread __pread
 pwrite
-rdchk _read __read _readdir __readdir
+_read __read _readdir __readdir
 _seekdir __seekdir
 select setenv setgidx setgroups setlocale setluid
 setmntent setpgid setpriv setsid setuidx
-- 
2.20.1

